handbook/onboarding/design.md

```diff
@@ -17,7 +17,13 @@ See [our Slack guide](/handbook/tools/slack.md) for more details.
 
 ### GitHub Teams
 
-Make sure you are a part of the GitHub team [`@ubclaunchpad/design`](https://github.com/orgs/ubclaunchpad/teams/design), and check that you have access to the [Design repository](https://github.com/ubclaunchpad/design) - this repository contains (some) design assets, resources, and tracking issues.
+Designers should have a presence in Github to know what is going on between product teams. First step would be to make a [Github](https://github.com/) account if you don't have one!
+
+Second, Make sure you are a part of the GitHub team [`@ubclaunchpad/design`](https://github.com/orgs/ubclaunchpad/teams/design), and check that you have access to the [Design repository](https://github.com/ubclaunchpad/design) - this repository contains (some) design assets, resources, and tracking issues.
+
+Third, make sure that you are added into your team's Github repository by your team's tech lead. 
```

Comment:
Should this be part of the checklist? I've left it out of the "Onboarding for Everyone" page because presumably this will be the leads' responsibility, and this onboarding will largely happen during/after team formation

Reply:
hmm yes I def see what you mean! yeah i guess we can remove it then!